File: src/main/java/com/gmail/borlandlp/minigamesdtools/gui/hotbar/items/example/ExampleItem.java
package com.gmail.borlandlp.minigamesdtools.gui.hotbar.items.example;

import com.gmail.borlandlp.minigamesdtools.Debug;
import com.gmail.borlandlp.minigamesdtools.gui.hotbar.items.SlotItem;
import org.bukkit.entity.Player;

public class ExampleItem extends SlotItem {
    @Override
    public boolean use(Player player) {
        Debug.print(Debug.LEVEL.NOTICE, "use item");
        return true;
    }
}

File: src/main/kotlin/com/gmail/borlandlp/minigamesdtools/gui/hotbar/items/example/ExampleItem.kt
package com.gmail.borlandlp.minigamesdtools.gui.hotbar.items.example

import com.gmail.borlandlp.minigamesdtools.Debug
import com.gmail.borlandlp.minigamesdtools.gui.hotbar.items.SlotItem
import org.bukkit.entity.Player

class ExampleItem : SlotItem() {
    override fun use(player: Player): Boolean {
        Debug.print(
            Debug.LEVEL.NOTICE,
            "use item"
        )
        return true
    }
}